conf/app.ini

```diff
@@ -55,6 +55,8 @@ DOMAIN = localhost
 ROOT_URL = %(PROTOCOL)s://%(DOMAIN)s:%(HTTP_PORT)s/
 HTTP_ADDR =
 HTTP_PORT = 3000
+; Permission for unix socket
+UNIX_SOCKET_PERMISSION = 0660
```

Comment:
Can confirm its not working... go-ini removes the leading zero, making it decimal 660, instead of octal 660